GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
 # Community Committee Governance
 
-The Community Committee governance is currently under pull request as the Community Committee Charter.
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
```

Comment:
Right, I think we're trying to reiterate as often as possible how important it is for the CC and the TSC to collaborate, participate, and engage in each other's efforts. Bringing important domain context into conversations that affect the project allows all of us to make -better- decisions.